client/packages/system/src/Location/ListView/ListView.tsx

```diff
@@ -50,7 +50,7 @@ const LocationListComponent: FC = () => {
       <AppBarButtons onCreate={() => onOpen()} />
       <DataTable
         id="location-list"
-        pagination={{ ...pagination, total: data?.totalCount }}
+        pagination={{ ...pagination, total: data?.totalCount ?? 0 }}
```

Comment:
🤔  there are 14 instances of the pagination being used like this, and this is the third which has the `?? 0`
Should we change the others?